gourmet/backends/db.py

```diff
@@ -1698,7 +1698,7 @@ def add_ing_to_keydic (self, item, key):
             self.do_add(self.keylookup_table,{'item':item,'ingkey':key,'count':1})
         # The below code should move to a plugin for users who care about ingkeys...
         for w in item.split():
-            w=str(w.lower())
+            w=str(w.decode('utf8').lower())
```

Comment:
* you can remove the string constructor, in the following line as well
* If `lower` is for case-insensitive comparisons, please us `casefold` https://docs.python.org/3/library/stdtypes.html?highlight=str#str.casefold

Reply:
Ok, I wasn't sure whether it's appropriate to do now, thanks!